en/gps_compass/rtk_gps_gem1305.md

```diff
@@ -0,0 +1,144 @@
+
+::: info
+
+- GEM1305 RTK doesn't support moving base mode yet.
+- GEM1305 RTK does not include a compass.
+
+:::
+
+## Where to Buy
+
+- [GEM1305 RTK receiver with antenna](https://www.datagnss.com/collections/gnss-for-drone/products/gem1305) (www.datagnss.com)
+
+  <img src="../../assets/hardware/gps/datagnss_gem1305/gem1305_hero.png" width="500px" alt="DATAGNSS GEM1305 RTK Receiver">
+
+- [DGM10 RTK Receiver](https://www.datagnss.com/collections/gnss-for-drone/products/dgm10-rtk-receiver) (same receiver in a casing).
+
+  ![dgm10_rtk_receiver.png](../../assets/hardware/gps/datagnss_gem1305/dgm10_rtk_receiver.png)
+
+## Key Features
+
+- Full constellation, multi-frequency GNSS satellite receiver
+- Support RTK with up to 10hz output rate
+- Standard UART serial interface
+- Lightweight only 50g or 26g(NANO RTK receiver)
+- High performance antenna
+
+## Frequencies
+
+- GPS/QZSS: L1 C/A, L5C
+- GLONASS: L1OF
+- BEIDOU: B1I, B2a
+- GALILEO: E1, E5a
+- IRNSS: L5
+
+## GNSS
+
+- 128 hardware channels
+- 3D accuracy: **1.5m** CEP
+- RTK accuracy: **2cm** +1PPM(H), 3cm+1PPM(V)
+
+## Interface
+
+- UART: 230400bps default
+- SMA connector for antenna
+- Output rate 5Hz default, up to 10Hz
+- Main power supply: 4.7~5.2V
+
+## Protocol
+
+- NMEA-0183 output
+- RTCMv3 input/output
+
+## Environment
+
+- Operating temp. -20~85°C
+
+## Dimension and Weight
+
+- 55x55x12mm
+- 50g (GEM1305) 26g (NANO with Helix)
+
+## Pinout
+
+The board is connected to the autopilot via UART interface.
+
+![GEM1305 connector](../../assets/hardware/gps/datagnss_gem1305/gem1305_connector.png)
+
+The 1.25mm pitch 6P connector (from left: PIN1 to PIN6):
+
+- 1: GND
+- 2: NC
+- 3: PPS
+- 4: Rx
+- 5: Tx
+- 6: 5V
+
+The board does not include a compass and so does not require connections for a compass part.
+
+## Hardware Setup
+
+RTK requires a base RTK module attached to the ground station, and a rover RTK module on the vehicle.
+The data from the base needs to be transmitted to the drone via telemetry radio and inputed into the RTK receiver on the rover.
+
+![RTK setup overview](../../assets/hardware/gps/datagnss_gem1305/setup_overview.png)
+
+The setup/connections for base and rover modules are shown below.
+
+### Base Setup (GCS)
+
+The base station connections are shown in the diagram below, along with the connection to a telemetry radio.
+
+![Base module setup](../../assets/hardware/gps/datagnss_gem1305/base_gnss_setup.png)
+
+Note that for the base we recommend the [NANO RTK Receiver](https://www.datagnss.com/collections/gnss-for-drone/products/multi-band-rtk-receiver-package) because it is easier to setup.
+
+<img src="../../assets/hardware/gps/datagnss_gem1305/nano_rtk_with_case.png" width="500px" alt="DATAGNSS NANO RTK Receiver">
+
+See to [How to setup Base station](https://wiki.datagnss.com/index.php/GEM1305-autopilot) for more details.
```

Comment:
This looks like a very complicated setup. Normally my instructions for RTK are "plug in the connected cable to USB, it should just work". This looks like a lot more than that.

What do you actually normally have to do?

Reply:
We just add the detail config for some messages output or disable output. If you use the fixed two device, one is for ground, another is for vehicle, it only need to be configed once time.

Reply:
Are you saying that you do need to follow those instructions, just once? (or that you will add more detailed instructions here). Sorry if I am being dumb.

Reply:
Yes, just once.
Next time just power up ,it will work.

Reply:
OK. FYI, I just improved the text a little. Instructions are fine other than ending up at mission planner, which doesn't work for PX4.